### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/layout/page.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#537] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/layout/page.html

图标支持 你可以在页面的 frontmatter 中配置 icon 字段，填入对应图标的 FontClass 即可绑定图标到页面。 该图标会在导航栏，侧边栏，路径导航和标题中使用。 例子 图标的相关设置，详见 图标支持 (../interface/icon.md) 路径导航 详见 路径导航 (breadcrumb.md) 章节。 文章信息展示 详见 文...

https://theme-hope.vuejs.press/zh/guide/layout/page.html

**Comment by gtwxxh666** [comment]
不知道为什么就不能显示了。
![](https://img.gtwxxh.cn/i/2023/04/644e635fe4f1e.webp)
主题配置没有设置为false

**Reply by gtwxxh666** [reply]
已解决

**Comment by leoleor** [comment]
由于 markdown.headers.level 的默认值是 [2, 3] ，因此 headerDepth 的默认最大值是 2 。这里是只有2、3级标题才能显示在侧边栏的意思吗

**Reply by Mister-Hope** [reply]
你可以提取更深层级标题以显示

**Reply by leoleor** [reply]
我设置了 headerDepth: 4，但是我的四级标题还是没显示出来

**Reply by Mister-Hope** [reply]
> 由于 markdown.headers.level 的默认值是 [2, 3] 
>
> 你可以提取更深层级标题

有什么理解上的问题？

**Reply by leoleor** [reply]
```  
markdown: {
    headers: {
      level: [2,3,4]
    }
  },
```
在config.ts里配置无效耶

**Reply by Mister-Hope** [reply]
Are you sure you are placing the options in the correct place (root of config.ts)?

**Reply by Mister-Hope** [reply]
If so give me a reproduction

**Comment by leoleor** [comment]
``` ts
import { searchPlugin } from "@vuepress/plugin-search";
import { defineUserConfig } from "vuepress";
import theme from "./theme.js";

export default defineUserConfig({
  base: "/leo2/",

  lang: "zh-CN",
  title: "一只攻城狮",
  description: "一只攻城狮",

  theme,

  markdown: {
    headers: {
      level: [2,3,4]
    }
  },
  
  plugins: [
    searchPlugin({
      // 你的选项
    }),
  ],

  // Enable it with pwa
  // shouldPrefetch: false,
});
```
这是config.ts文件

**Reply by rgkjhshi** [reply]
> 在桌面模式下，文章的标题列表会自动显示在屏幕的右侧。

标题列表如何显示在右侧？现在是最上面了

**Reply by sttgood** [reply]
同问

**Comment by rgkjhshi** [comment]
> 在桌面模式下，文章的标题列表会自动显示在屏幕的右侧。
标题列表在上方还是在右侧显示，能不能给个配置啊？时而在上方，时而在右侧，真的很扯～

**Comment by rgkjhshi** [comment]
> 在桌面模式下，文章的标题列表会自动显示在屏幕的右侧。

标题列表在上方还是在右侧显示，能不能给个配置啊？时而在上方，时而在右侧，真的很扯～

**Reply by Mister-Hope** [reply]
> > 在桌面模式下，文章的标题列表会自动显示在屏幕的右侧。
> 
> 标题列表在上方还是在右侧显示，能不能给个配置啊？时而在上方，时而在右侧，真的很扯～

布局只跟网页横向像素数量有关，所以如果你一直在最大化/全屏使用网页，那么一个能够来回改变自己的物理像素的显示器或者来回改变系统分辨率的行为，也

## 真的很扯～

**Reply by rgkjhshi** [reply]
抱歉是我表达的问题，我是想说 标题列表 在布局中的位置是否可以固定下来，并且可以通过配置来控制显示的位置，而不是自动调整位置
